src/server/db_migrations/20250721_add_urls_safebrowsing_expiry.sql

```diff
@@ -0,0 +1,11 @@
+-- This adds the safeBrowsingExpiry column to the urls table, which is
+-- backwards-compatible with the current codebase as it allows NULL values.
+
+BEGIN TRANSACTION;
+
+ALTER TABLE urls ADD "safeBrowsingExpiry" TIMESTAMP WITH TIME ZONE;
+
+COMMIT;
+
+-- Down migration
+-- ALTER TABLE urls DROP COLUMN "safeBrowsingExpiry";
```

Comment:
question: how will we run this migration since it looks like we're not using `sequelize-cli`?

Reply:
from sequelize-cli [docs](https://github.com/sequelize/cli): `sequelize migration:generate` -> `sequelize migration:generate --name migration-name`

Reply:
Updated in db085469f146dfcc4d40408ef9bdf49c87b61d99.